### PR TITLE
parser: upload tidb version

### PR DESCRIPTION
**Pull request by leoppro**
upload tidb version to pingcap/tidb#971629b9477a

### CODE REVIEW

**Comment by leoppro** [comment]
@kennytm @XuHuaiyu PTAL

File: go.sum1 [hunk 1]
```diff
@@ -60,6 +59,7 @@ github.com/golang/groupcache v0.0.0-20181024230925-c65c006176ff h1:kOkM9whyQYodu
 github.com/golang/groupcache v0.0.0-20181024230925-c65c006176ff/go.mod h1:cIg4eruTrX1D+g88fzRXU5OdNfaM+9IcxsU14FzY7Hc=
 github.com/golang/lint v0.0.0-20180702182130-06c8688daad7/go.mod h1:tluoj9z5200jBnyusfRPU2LqT6J+DAorxEvtC7LHB+E=
 github.com/golang/mock v1.1.1/go.mod h1:oTYuIxOrZwtPieC+H1uAHpcLFnEyAGVDL/k47Jfbm0A=
+github.com/golang/protobuf v0.0.0-20180814211427-aa810b61a9c7/go.mod h1:6lQm79b+lXiMfvg/cZm0SGofjICqVBUtrP5yJMmIC1U=
```
Comment (XuHuaiyu) [inline go.sum1]: Why do these line also changed?

Comment (XuHuaiyu) [reply go.sum1]: @tiancaiamao PTAL

**Comment by amyangfei** [comment]
LGTM

File: go.sum1 (hunk 1 above)
Comment (kennytm) [reply go.sum1]: I think this is just a side effect of upgrading TiDB.

File: go.mod1 [hunk 2]
```diff
@@ -7,12 +7,10 @@ require (
 	github.com/cznic/sortutil v0.0.0-20150617083342-4c7342852e65
 	github.com/cznic/strutil v0.0.0-20171016134553-529a34b1c186
 	github.com/cznic/y v0.0.0-20170802143616-045f81c6662a
-	github.com/pingcap/check v0.0.0-20181213055612-5c2b07721bdb
+	github.com/pingcap/check v0.0.0-20190102082844-67f458068fc8
 	github.com/pingcap/errors v0.11.0
-	github.com/pingcap/tidb v0.0.0-20181217070741-096bb68e6bef
+	github.com/pingcap/tidb v0.0.0-20180108134023-971629b9477a
```
Comment (kennytm) [inline go.mod1]: Why is the date 201<b>8</b>0108134023 🤔

Comment (amyangfei) [reply go.mod1]: git commit hash is right, ref to https://github.com/pingcap/tidb/commit/971629b9477a9027f30e83f1fdb17120c0f2bec1 . strange

Comment (leoppro) [reply go.mod1]: it is because of the bug in `gopkg.in/stretchr/testify.v1@v1.3.0`

**Pull request merged**